addons/account_peppol/wizard/service_wizard.py

```diff
@@ -0,0 +1,109 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+from markupsafe import Markup
+
+from odoo import api, fields, models, _
+
+_logger = logging.getLogger(__name__)
+
+
+class PeppolService(models.TransientModel):
+
+    _name = 'account_peppol.service'
+    _order = 'document_name'
+    _description = 'Peppol Service'
+
+    wizard_id = fields.Many2one('account_peppol.service.wizard')
+    document_identifier = fields.Char()
+    document_name = fields.Char()
+    enabled = fields.Boolean(default=False)
+
+
+class PeppolServiceConfig(models.TransientModel):
+
+    _name = 'account_peppol.service.wizard'
+    _description = 'Peppol Services Wizard'
+
+    edi_user_id = fields.Many2one('account_edi_proxy_client.user', string='EDI user')
+    service_json = fields.Json(help="JSON representation of peppol services as retrieved from the peppol server.")
+    service_info = fields.Html(compute='_compute_service_info')
+    service_ids = fields.One2many(
+        comodel_name='account_peppol.service',
+        inverse_name='wizard_id',
+        readonly=False,
+    )
+
+    # -------------------------------------------------------------------------
+    # COMPUTES
+    # -------------------------------------------------------------------------
+
+    def _compute_service_info(self):
+
+        for wizard in self:
+            message = ''
+            supported_doctypes = self.env['res.company']._peppol_supported_document_types()
+            if (non_configurable := [
+                identifier for identifier in (wizard.service_json or {})
+                if identifier not in supported_doctypes
+            ]):
```

Comment:
This is barely understandable ...
```suggestion
            if (non_configurable := [
                identifier
                for identifier in (wizard.service_json or {})
                if identifier not in supported_doctypes
            ]):
```